### ISSUE TITLE
revise language on increments

### ISSUE THREAD

**Issue by teixeirak** [forc-db/IPCC-EFDB-integration#48]
current text:
In the case of forests, emission factors include C stocks, **net increments (“stock changes”)**, and fluxes (“gains” and “losses”) for various pools [(IPCC, ](https://github.com/forc-db/IPCC-EFDB-integration/issues/new#_bookmark33)[2006, ](https://github.com/forc-db/IPCC-EFDB-integration/issues/new#_bookmark33)[2019a).](https://github.com/forc-db/IPCC-EFDB-integration/issues/new#_bookmark34)

Sandro's comment:
This is to be revised. The "net increment" in aboveground is the "gain" element of the gain and loss method. The net increment is the gross increment minus mortality (excluding mortality associated with disturbances)

**Comment by teixeirak** [comment]
current text:
With input from the IPCC’s Technical Support Unit, we reviewed the list of ForC variables to identify those that were relevant to EFDB and to appropriately map them into EFDB (Fig. 1). For each C pool (Table 1), we identified variables representing organic matter or C stocks, stock changes (a.k.a. “net annual increments” by IPCC, “increments” in ForC), gains (a.k.a. “gross annual increments” by IPCC, “fluxes” in ForC), and losses (“fluxes” in ForC). 

Sandro's comment:
This is not correct. The biomass increment to be reported as a gain in the ipcc methodology is the net-increment (i.e. gross increment minus mortality). [where mortality does not include mortality caused by disturbances, which is otherwise to be reported as C stock losses]

**Comment by teixeirak** [comment]
Summarizing:

net increment  = "gain" element of the gain and loss method
net increment =  gross increment - mortality (excluding mortality associated with disturbances)

**Comment by teixeirak** [comment]
Revised the two sections of concern as follows: 

**In the case of forests,emission factors include C stocks, net annual increments, and annual fluxes for various pools (IPCC, 2006, 2019a).

For each C pool (Table 1), we identified variables representing organic matter or C stocks, net annual increments, influxes (a.k.a. "gross annual increments" by IPCC), and outfluxes.**

I just wrote to Sandro to confirm that these are correct. Unless I hear back otherwise, I will assume this issue is resolved.
